src/Player.ts

```diff
@@ -130,6 +134,18 @@ class Player {
       selecthotkey: 0,
       esc: 0,
     };
+    this.hotkeys = {
```

Comment:
Can we rename that variable to groupHotkeys so we know it is just about the group assignment/usage?